core/components/atoms/select/select.js

```diff
@@ -19,6 +19,9 @@ const Select = ({ options, ...props }) => {
 
   return (
     <StyledSelect {...props} {...Automation('select')}>
+      <option disabled selected={!props.value} value="">
```

Comment:
what does `!props.value` do 🤔

Reply:
If we have a `props.value` set, this won't be the selected option, the `value` value will be

Reply:
😅 got it

can you leave a comment there, looks like it has something to do with boolean values !false = true

Reply:
Sure thing! Done.